modin/core/storage_formats/pandas/parsers.py

```diff
@@ -308,23 +308,21 @@ def parse(chunks, **kwargs):
         num_splits = kwargs.pop("num_splits", None)
         index_col = kwargs.get("index_col", None)
 
+        kwargs_bio = kwargs.copy()
+        compression = kwargs_bio.pop("compression", "infer")
+        storage_options = kwargs_bio.pop("storage_options", None) or {}
         pandas_dfs = []
         for fname, start, end in chunks:
             if start is not None and end is not None:
                 # pop "compression" from kwargs because bio is uncompressed
```

Comment:
please also move this comment about popping "compression" from kwargs

Reply:
done